packages/@uppy/dashboard/src/utils/createSuperFocus.test.js

```diff
@@ -0,0 +1,10 @@
+const createSuperFocus = require('./createSuperFocus')
+
+describe('createSuperFocus', () => {
+  // superFocus.cancel() is used in dashboard
+  it('should return a function that can be cancelled', () => {
```

Comment:
Confused, sorry, but where is `superFocus.cancel()` defined?

Reply:
@arturi, it's defined on the function that's returned from `debounce()`!
(Thank you for starting to review this PR, I'll answer all comments when I get some more time)